src/derivatives.jl

```diff
@@ -36,7 +36,8 @@ function jacobian(::Type{RotMatrix},  q::Quat)
     # then R = RotMatrix(q) = RotMatrix(s * qhat) = s * RotMatrix(qhat)
 
     # get R(q)
-    R = q[:]
```

Comment:
For future reference, this is the only place where https://github.com/JuliaArrays/StaticArrays.jl/issues/128 was causing issues.